plugins/env/app/controllers/admin/environment_variable_groups_controller.rb

```diff
@@ -34,10 +34,10 @@ def destroy
     def preview
       @groups = if params[:group_id]
         @group = EnvironmentVariableGroup.find(params[:group_id])
-        SamsonEnv.env_groups(Stage.new(environment_variable_groups: [@group], deploy_groups: DeployGroup.all))
+        SamsonEnv.env_groups(Stage.new(project: Project.new(environment_variable_groups: [@group]), deploy_groups: DeployGroup.all))
```

Comment:
This signature is becoming kinda complicated, If you were starting from scratch would it still take a stage? Maybe a project and an array or splat of deploy groups?